index.html

```diff
@@ -4,9 +4,66 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Hello, YOUR_NAME</title>
+  <title>Hello, Pat</title>
+  <style>
+    header {
+      background: #053;
+      margin: 0 auto;
+    }
+    h1 {
+      font-size: 32px;
+      padding: 20px 0 0 0;
+      font-style: italic;
+    }
+    h2 {
+      font-size: 22;
+      padding: 30px;
+    }
+    body {
+      background: rgb(45, 128, 191);
+      text-align: center;
+      font-family: cambria;
+      color: white;
+    }
+    div {
+      padding: 10px;
+      margin: 50px auto;
+      background: rgb(36, 187, 165);
+      max-width: 900px;
+      text-align: center;
+    }
+    p {
+      font-size: 22px;
+    }
+    h4 {
+      font-size: 28;
+      color:
+    }
+  </style>
 </head>
+
 <body>
-  <!-- use this space to add your own content -->
+  <header>
+    <h1> Hi, I'm Pat Sullivan </h1>
+    <h2> I am a Junior and am majoring in PPE - Philosophy, Politics, and Economics - through the Center for Interdisciplinary Studies <br /> <small> This semester I took DIG 245 and learned about web development </h2>
+  </header>
+  <div>
+    <h3> This course </h3>
+    <p> I am taking this course because I want to learn how to make effective and visually-appealing web pages and applications. I have always wanted to know how web pages work internally, as I use them every day.</p>
+  </div>
+  <div>
+    <h3>My experience</h3>
+    <p> I've learned how to write and style a page using HTML and CSS. I can <code>manipulate the text</code> and layout a web page using HTML tags. I can then style my page using CSS, changing the size, shape, color, background, and spacing of the objects I've created in HTML.</p>
+  </div>
+  <div>
+    <h3> Some Cool Web Projects </h3>
+    <h4><a href="https://ncov2019.live/data" target="_blank"> ncov2019.live Global Corona Virus tracker </a></h4>
```

Comment:
Structurally, this is a list. I might mark it up as something like:

```html
<div id="cool-projects">
  <h3>Cool Projects</h3>
  <ol class="project-list">
    <li>
      <h4><a href="...">ncov2019.live...</a></h4>
      <p>This website etc. etc.</p>
    </li>
    <li>
      <h4><a href="...">ncov2019.live...</a></h4>
      <p>This website etc. etc.</p>
    </li>
    <li>
      <h4><a href="...">ncov2019.live...</a></h4>
      <p>This website etc. etc.</p>
    </li>
  </ol>
</div>
```

Then in CSS you can select + style the relevant portions with selectors like...

```css
.project-list li h4 {
  font-size: ...;
}

.project-list li h4 a {
  color: ...;
}
```

"But Jesse," you might say, "I don't want those bullets."

CSS lets you control almost all aspects of display: https://www.w3schools.com/css/css_list.asp

Reply:
The main thing is to wrap sections in names (so you can reference them) and mark things up as they're structured so you can select them, e.g., "Every list item in the blah-de-blah section."